### PR TITLE
Document available time types and their formats more explicitly

### PR DESCRIPTION
**Pull request by capnrefsmmat**
List the formats and how to interpret them. List available time types for every signal so users know what to request (since they *must* specify a time type to fetch a signal).

### CODE REVIEW

File: docs/api/covidcast_times.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+nav_order: 5
+---
+
+# Date Coding and Revisions
+
+Every observation in the COVIDcast Epidata API has two dates attached:
+
+* `time_value`: The time the underlying events happened. For example, for a data
+  source that reports on COVID test results, the time value is the date the
+  results were recorded by the testing provider.
```
Comment (Akvannortwick) [inline docs/api/covidcast_times.md:13]:
```suggestion
* `time_value`: The time the underlying events happened. For example, when a data
  source reports on COVID test results, the time value is the date the
  results were recorded by the testing provider.
```

File: docs/api/covidcast_times.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* `issue`: The date the estimates were *issued*. For example, a COVID test
+  result might be recorded on October 1st, but it may take several days for
+  that report to be collected, aggregated, received by Delphi, and added to our
+  database. The date Delphi makes the data available is the *issue date*.
```
Comment (Akvannortwick) [inline docs/api/covidcast_times.md:17]:
```suggestion
  database.  The *issue date* is when Delphi makes the data available.
```

File: docs/api/covidcast_times.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+For example, consider using our [doctor visits signal](covidcast-signals/doctor-visits.md),
+which estimates the percentage of outpatient doctor visits that are
+COVID-related, and consider a result row with `time_value = "2020-05-01"` for
+`geo_value = "pa"`. This is an estimate for the percentage in Pennsylvania on
+May 1, 2020. That estimate was issued on May 5, 2020, the delay being due to the
```
Comment (Akvannortwick) [inline docs/api/covidcast_times.md:23]:
```suggestion
May 1, 2020. That estimate was issued on May 5, 2020, the delay is due to the
```

**Review by Akvannortwick — COMMENTED** [review]
Some minor suggestions otherwise it looks good.

File: docs/api/covidcast_times.md (hunk 3 above)
Comment (capnrefsmmat) [reply docs/api/covidcast_times.md:23]: This change creates a comma splice; we'd either have to make it two sentences or leave it as-is

Comment (krivard) [reply docs/api/covidcast_times.md:23]: how about

```suggestion
May 1, 2020, which was issued on May 5, 2020. The delay is due to the
```

**Pull request merged**